prov/rxm/src/rxm.h

```diff
@@ -956,6 +957,29 @@ static inline int rxm_cq_write_recv_comp(struct rxm_rx_buf *rx_buf,
 				    rx_buf->pkt.hdr.tag);
 }
 
+static inline enum fi_hmem_iface rxm_iov_to_hmem_iface(const struct iovec *iov,
+						       size_t count)
+{
+	if (!count || !iov)
+		return FI_HMEM_SYSTEM;
+	return ofi_get_hmem_iface(iov[0].iov_base);
+}
+
+static inline enum fi_hmem_iface rxm_init_hmem_iface(struct rxm_ep *rxm_ep,
```

Comment:
Name needs help.  This is determining the hmem interface.

Reply:
This patch will be removed.